.github/workflows/ci.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         id: generate-python-key
         run: >-
           echo "::set-output name=key::base-venv-${{ env.CACHE_VERSION }}-${{
-            hashFiles('pylint/__pkg_info__.py', 'requirements_test.txt',
+            hashFiles('pylint/__pkg_info__.py', 'requirements_test.txt', 'setup.cfg',
```

Comment:
Since `pylint/__pkg_info__.py` doesn't contain the requirements anymore, it isn't necessary and can be removed here. Just so it's the same for all `hashFiles` calls, would you mind putting `setup.cfg` at the first position (the one `__pkg_info__.py` has right now)?

Reply:
There's still the version in ``__pkg_info__.py``, or was it just for the requirements ?

Reply:
It was just for the requirements. So should be safe to remove.